tool/dev.dart

```diff
@@ -32,5 +33,8 @@ Future<Null> main(List<String> args) async {
   config.copyLicense.directories = directories;
   config.format.paths = directories;
 
+  config.test
+    ..platforms = Platform.version.startsWith('2') ? ['chrome'] : ['vm'];
```

Comment:
In Dart 1 we run tests on the vm?

Reply:
Currently I believe we are. So I thought I'd leave it on the VM. I could change that to content shell if that would be preferable. I don't have a strong feeling about it.